dart/test/protocol/rate_limiter_test.dart

```diff
@@ -166,6 +166,25 @@ void main() {
     final result = rateLimiter.filter(envelope);
     expect(result, isNull);
   });
+
+  test(
+      'When no sentryRateLimitHeader available, it fallback to retryAfterHeader',
+      () {
+    final rateLimiter = fixture.getSUT();
+    fixture.dateTimeToReturn = 0;
+    final eventItem = SentryEnvelopeItem.fromEvent(SentryEvent());
+    final envelope = SentryEnvelope(
+      SentryEnvelopeHeader.newEventId(),
+      [eventItem],
+    );
+
+    rateLimiter.updateRetryAfterLimits(null, '50', 429);
```

Comment:
The test has the same expectation as the previous one. Is there an easy way to test the case where we don't have a null result after filter?

Reply:
indeed but it's passing a value now so it is still worth testing.